modules/dr-restoring-cluster-state-about.adoc

```diff
@@ -1,10 +1,13 @@
 // Module included in the following assemblies:
 //
 // * disaster_recovery/scenario-2-restoring-cluster-state.adoc
+// * etcd/etcd-backup.adoc
 
 :_mod-docs-content-type: CONCEPT
 [id="dr-scenario-2-restoring-cluster-state-about_{context}"]
-= About restoring cluster state
+= Restoring to a previous cluster state
```

Comment:
Somewhat mentioned this earlier for another section, but just an FYI that using a gerund phrase in the title of this concept module might make it seem like this section will have a procedure, at least by the logic of our mod docs guidelines

Reply:
Leaving as-is for the purposes of this PR to migrate the content, but I'll take a note to address in a future PR.